src/modules/network.cpp

```diff
@@ -57,6 +57,7 @@ namespace modules {
     m_accumulate = m_conf.get(name(), "accumulate-stats", m_accumulate);
     set_interval(1s);
     m_unknown_up = m_conf.get<bool>(name(), "unknown-as-up", false);
+    m_iec = m_conf.get<bool>(name(), "iec", m_iec);
```

Comment:
Maybe the config option should be called `si-units` or `metric-units` and `true` means `1KB = 1000B`, which is probably clearer than IEC to many people.